aea/cli/fingerprint.py

```diff
@@ -35,14 +35,24 @@
     CONNECTION,
     CONTRACT,
     DEFAULT_CONNECTION_CONFIG_FILE,
+    DEFAULT_CONTRACT_CONFIG_FILE,
     DEFAULT_PROTOCOL_CONFIG_FILE,
     DEFAULT_SKILL_CONFIG_FILE,
     PROTOCOL,
     SKILL,
 )
+from aea.configurations.data_types import PackageType
 from aea.configurations.loader import ConfigLoader
 
 
+CONFIG_FILE_TO_PACKAGE_TYPE = {
```

Comment:
This would have already been defined elsewhere I believe. If not, define in configurations.constants and import here